core/src/test/java/com/graphhopper/routing/AlternativeRouteTest.java

```diff
@@ -69,28 +71,28 @@ public AlternativeRouteTest(TraversalMode tMode) {
         });
     }
 
-    public static void initTestGraph(Graph graph) {
+    public static void initTestGraph(Graph graph, FlagEncoder encoder) {
         /* 9
          _/\
          1  2-3-4-10
          \   /   \
          5--6-7---8
         
          */
-        graph.edge(1, 9, 1, true);
-        graph.edge(9, 2, 1, true);
-        graph.edge(2, 3, 1, true);
-        graph.edge(3, 4, 1, true);
-        graph.edge(4, 10, 1, true);
+        GHUtility.setProperties(graph.edge(1,9).setDistance(1),encoder,60,true, true);
```

Comment:
We should probably run code formatting here. For example you can undo all changes and commit them again with the "format  on commit" checkbox enabled in intellij.

Reply:
Ok, will do. (I usually do not use IntelliJ for git)

Reply:
I just meant this is an easy way to apply the formatting to the changed files only. I think we never fixed the formatting for the entire project so its hard to run it everywhere ;)